fiftyone/factory/repos/delegated_operation_doc.py

```diff
@@ -68,6 +69,7 @@ def from_pymongo(self, doc: dict):
             doc["completed_at"] if "completed_at" in doc else None
         )
         self.failed_at = doc["failed_at"] if "failed_at" in doc else None
+        self.pending_at = doc["pending_at"] if "pending_at" in doc else None
         self.pinned = doc["pinned"] if "pinned" in doc else None
         self.dataset_id = doc["dataset_id"] if "dataset_id" in doc else None
         self.run_link = doc["run_link"] if "run_link" in doc else None
```

Comment:
nit: I agree with a suggestion from coderabbit, can we just simply replace all of these `doc["thing"] if "thing" in doc else None` with `doc.get("thing")`

Reply:
Agree that the 'if thing in doc' syntax should be simplified to just 'get' but is it possible doc is None and if so, should it throw? If not maybe what's needed here is just a 'if doc'?

Reply:
I don't think it's possible. It would throw before and it would throw now